dependencies.yaml

```diff
@@ -327,10 +327,18 @@ dependencies:
           - matrix:
               cuda: "12.*"
             packages:
+              - cuda-cudart-dev
+              - cuda-nvtx-dev
+              - cuda-profiler-api
+              - libcublas-dev
+              - libcurand-dev
+              - libcusolver-dev
+              - libcusparse-dev
```

Comment:
Just curious, why  packages other than  `cuda-nvtx-dev` needed here?

Reply:
They're build dependencies of cugraph. These are listed in the conda recipes. However, they've been coming transitively as dependencies of libraft. We should be explicit about the libraries that are used rather than relying on them coming from raft.